spec.html

```diff
@@ -37,6 +37,165 @@ <h1>Introduction</h1>
+      <p>The _AggregateError_ constructor:</p>
+      <ul>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error%.</li>
+        <li>has a `name` property whose value is the String value `"AggregateError"`.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype">
+        <h1>AggregateError.prototype</h1>
+        <p>The initial value of `AggregateError.prototype`  is the intrinsic object %AggregateErrorPrototype%.</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregate-error-prototype-objects">
+      <h1>Properties of the _AggregateError_ Prototype Object</h1>
+      <p>The _AggregateError_ prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype.constructor">
+        <h1>_AggregateError_.prototype.constructor</h1>
+        <p>The initial value of `AggregateError.prototype.constructor` is the intrinsic object %_AggregateError_% (<emu-xref href="#sec-aggregate-error-constructors"></emu-xref>).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.name">
+        <h1>_AggregateError_.prototype.name</h1>
+        <p>The initial value of `AggregateError.prototype.name` is `"AggregateError"`.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.errors">
+        <h1>_AggregateError_.prototype.errors</h1>
+        <p>The initial value of `AggregateError.prototype.errors` is an empty Array.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.toString">
+        <h1>_AggregateError_.prototype.toString ( )</h1>
+        <p>The following steps are taken:</p>
+        <emu-alg>
+          1. Let _O_ be the *this* value.
+          1. If Type(_O_) is not Object, throw a *TypeError* exception.
+          1. Let _name_ be ? Get(_O_, `"name"`).
+          1. If _name_ is *undefined*, set _name_ to `"AggregateError"`; otherwise set _name_ to ? ToString(_name_).
+          1. Let _msg_ be ? Get(_O_, `"message"`).
+          1. If _msg_ is *undefined*, set _msg_ to the empty String; otherwise set _msg_ to ? ToString(_msg_).
+          1. Let _errorsNum_ be ? ToLength(? Get(_O_, "errors")).
+          1. Let _errorsNumString_ ! ToString(_errorsNum_).
+          1. If _name_ is the empty String, return _msg_.
+          1. If _msg_ is the empty String, return _name_.
+          1. Return the string-concatenation of _name_, the code unit 0x003A (COLON), the code unit 0x0020 (SPACE), _msg_, "(", _errorsNumString_, " messages)".
```

Comment:
This results in `"1 messages"`.

FWIW, I'd be totally okay with leaving this up to implementations, by making it part of the `message` property. We could have a non-normative note that recommends implementers to include the message count in the error message.